src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -188,7 +188,7 @@ public static function create($params = [])
          * Select an appropriate value for $term_id.
          */
         $term_id = MyRadio_Term::getActiveApplicationTerm()->getID();
-	$num_weeks = MyRadio_Term::getActiveApplicationTerm()->getTermWeeks();
+	    $num_weeks = MyRadio_Term::getActiveApplicationTerm()->getTermWeeks();
```

Comment:
?

Reply:
not actually sure why there is a tabspace. I think added a debug print and removed it but in my editor it hasn't got any tabspaces

Reply:
![image](https://github.com/user-attachments/assets/e999bdda-f92d-43db-88fe-1e8de28feeb7)